cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflow.java

```diff
@@ -106,4 +106,12 @@ protected final void addMapReduce(String mapReduce) {
   protected final void addSpark(String spark) {
     configurer.addSpark(spark);
   }
+
+  /**
+   * Adds a fork to the {@link Workflow}
```

Comment:
I think we only add full stop if the comment is for class, or it is for method and has multiple lines.